dataland-frontend/tests/e2e/specs/community/DataRequest.ts

```diff
@@ -58,33 +53,51 @@ describeIf(
         checkIfIdentifiersProperlyDisplayed(interception);
       });
 
-      cy.get('[data-test="submittingSuccededMessage"] [data-test="someIdentifiersPassed"]')
+      cy.get('[data-test="acceptedIdentifiers"]')
         .should("exist")
-        .get("p")
-        .contains("All identifiers have been submitted successfully.");
+        .get('[data-test="identifiersHeading"')
+        .contains("1 REQUESTED IDENTIFIER");
+
+      cy.get('[data-test="requestStatusText"]').should("exist").contains("Success");
     });
 
     it("When identifiers are rejected", () => {
-      cy.intercept("POST", "**/community/requests").as("postRequestData");
-
       checksBasicValidation();
-      choseFramewors();
+      chooseFrameworks();
 
       cy.get("textarea[name='listOfCompanyIdentifiers']")
         .type("12345incorrectNumber")
         .get("button[type='submit']")
         .should("exist")
         .click();
 
-      cy.get('[data-test="failMessage"]')
+      cy.get('[data-test="selectedIdentifiersUnsuccessfulSubmit"]')
         .should("exist")
-        .contains("All provided company identifiers have an invalid format.");
+        .get('[data-test="identifiersHeading"')
+        .contains("SELECTED IDENTIFIERS");
+
+      cy.get('[data-test="requestStatusText"]').should("exist").contains("Request Unssuccessful");
     });
 
+    /**
+     * Chose reporting periods
```

Comment:
Typo

Reply:
Is there anything you wouldn't find? 😀 Thanks, done!

Reply:
Thank you and closed